test/cypress/integration/getting_started_spec.ts

```diff
@@ -332,6 +306,27 @@ describe('Getting started modal', () => {
         });
     });
 
+    it('only enables one environment to be selected at a time', () => {
```

Comment:
Maybe:
```suggestion
    it('should allow only one environment to be selected at a time', () => {
```
would be more accurate in this context ("allow" sounds for me better suited than "enable") as well as be consistent with other tests naming?

Reply:
I agree, that looks more suitable :)
I'll provide that inside the upcoming commit

Reply:
Done